kogito-editors-js/packages/boxed-expression-component/showcase/src/index.tsx

```diff
@@ -36,6 +36,7 @@ import { Button, Modal } from "@patternfly/react-core";
 import { CopyIcon, PenIcon } from "@patternfly/react-icons";
 import "./lib/components/BoxedExpressionEditor/base-no-reset-wrapped.css";
 import ReactJson from "react-json-view";
+import { dataTypeProps, pmmlParams } from "./lib/__tests__/__mocks__";
```

Comment:
In my opinion the showcase belongs to the testing world more than the production one. So it is fine to have a dependency (a simple constant) from it.
I am excluding to duplicate things because it is important to share the same data type structure for both the testing and showcase, as an expectation of the input in the production case.
Please let me know if you have a better proposal for where the constant to be present. Didn't find a better location.

Reply:
Makes sense. I can't think of a better place to place it too.